crowdsec-docs/unversioned/troubleshooting/issue_lp_no_alerts.md

```diff
@@ -69,7 +69,7 @@ sudo cscli alerts list --origin <machine-name>
 
 ### If no logs are being read
 
-Follow the [LP No Logs Read troubleshooting guide](/u/troubleshooting/issue_lp_no_logs_read) for detailed steps.
+Follow the [Log Processor No Logs Read troubleshooting guide](/u/troubleshooting/issue_lp_no_logs_read) for detailed steps.
```

Comment:
Identify the affected Log Processor -> already indicated by yhe console
**FOR ALL NO ALERTS LP/SE** : point to healthcheck scenarios